abe/cpabe/tkn20/example_test.go

```diff
@@ -50,7 +50,7 @@ func checkPolicy(in map[string][]string) bool {
 
 func Example() {
 	policyStr := `(occupation: doctor) and (country: US)`
-	invalidPolicyStr := `(ocupation: doctor) and (country: pacific)`
+	invalidPolicyStr := `(occupation1: doctor) and (country: pacific)`
```

Comment:
This one is odd. I'm pretty sure the goal is to recognize something that doesn't have `occupation`, I think that `occupation_` or `occupation1` will work just as well as `ocupation` without resulting in spell checking tools complaining. Happy to drop this (or anything else).

Reply:
Sounds good, applied as https://github.com/cloudflare/circl/pull/456/commits/7cdef5cf5c8f2e08367e8a35b42733369a3488f3